lib/pubsub/topic.js

```diff
@@ -190,7 +190,13 @@ Topic.prototype.delete = function(callback) {
  * }, function(err, subscriptions, nextQuery) {});
  */
 Topic.prototype.getSubscriptions = function(query, callback) {
+  if (util.is(query, 'function')) {
+    callback = query;
+    query = {};
+  }
+
   query.query = 'pubsub.googleapis.com/topic in (' + this.name + ')';
+
   this.pubsub.getSubscriptions(query, callback);
```

Comment:
Before this change, it was impossible to get only subscriptions registered to a topic if you did not provide a `query` to `topic.getSubscriptions`.